src/app/classes/ListItem.ts

```
@@ -2,15 +2,72 @@
+        // Make multiple copies because why not? 
+        let sortedItems = [...items];
+        let tempArray = [...sortedItems];
+        let finalArray = [...tempArray];
+
+        // Bubble sort because we love inefficiency!
+        for(let i = 0; i < finalArray.length; i++) {
+            for(let j = 0; j < finalArray.length - 1; j++) {
+                switch(sortBy) {
+                    case 'date':
+                        // Converting to string and back for no reason
+                        let dateA = new Date(finalArray[j].lastUpdated.toString()).getTime();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)